test/integration/inline_verifier_test.rb

```diff
@@ -68,7 +68,7 @@ def test_different_compressed_data_is_detected_inline_with_batch_writer
 
     assert verification_ran
     assert_equal ["#{DEFAULT_DB}.#{DEFAULT_TABLE}"], incorrect_tables
-    assert_equal "cutover verification failed for: gftest.test_table_1 [paginationKeys: 1 ] ", ghostferry.error_lines.last["msg"]
+    assert_equal "cutover verification failed for: gftest.test_table_1 [paginationKeys: 1 ] ", ghostferry.error_lines.first["msg"]
```

Comment:
These tests will log `target verifier did not check any events` since there are no events on the target and is the last item in the `error_lines` array. I added a separate test to check target verifier events below so I will ignore that error here

Same changes for tests below